geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/parameters/SpopParameterRequirements.java

```diff
@@ -18,18 +18,22 @@
 import static org.apache.geode.redis.internal.RedisConstants.ERROR_NOT_INTEGER;
 import static org.apache.geode.redis.internal.netty.Coder.bytesToLong;
 
+import java.util.function.BiConsumer;
+
 import org.apache.geode.redis.internal.netty.Command;
 import org.apache.geode.redis.internal.netty.ExecutionHandlerContext;
 
-public class SpopParameterRequirements implements ParameterRequirements {
-  @Override
-  public void checkParameters(Command command, ExecutionHandlerContext context) {
-    if (command.getProcessedCommand().size() == 3) {
-      try {
-        bytesToLong(command.getProcessedCommand().get(2));
-      } catch (NumberFormatException nex) {
-        throw new RedisParametersMismatchException(ERROR_NOT_INTEGER);
+public class SpopParameterRequirements {
```

Comment:
Possibly out of scope for this PR, but this class probably shouldn't exist, and the check that it's doing to validate arguments being integers should be moved to the SPopExecutor.

Reply:
Good point - I've made that change.